app/src/main/java/org/projectbuendia/client/models/AppModel.java

```diff
@@ -301,22 +301,13 @@ public FetchTypedCursorAsyncTask(
         }
 
         @Override protected TypedCursor<T> doInBackground(Void... voids) {
-            Cursor cursor = null;
-            try {
-                cursor = mContentResolver.query(
-                    mContentUri,
-                    mProjection,
-                    mFilter.getSelectionString(),
-                    mFilter.getSelectionArgs(mConstraint),
-                    null);
-
+            try (Cursor cursor = mContentResolver.query(
+                mContentUri,
+                mProjection,
+                mFilter.getSelectionString(),
+                mFilter.getSelectionArgs(mConstraint),
+                null)) {
```

Comment:
FYI, try-with-resources requires API level 19 (KitKat). I think we should use it for this project (where we control the hardware) but just wanted to let you know for the future. :)

The `try{} finally{c.close()}` pattern is what you should use if you need to support pre-KitKat.